crates/store/re_types/definitions/rerun/blueprint/archetypes/visual_bounds2d.fbs

```diff
@@ -16,4 +16,7 @@ table VisualBounds2D (
     ///
     /// Use this to control pan & zoom of the view.
     range: rerun.blueprint.components.VisualBounds2D ("attr.rerun.component_required", order: 1000);
+
+    /// Controls the distance to the near clipping plane
+    clipping_plane: rerun.blueprint.components.ClippingPlane ("attr.rerun.component_optional", order: 2000);
```

Comment:
maybe better to call the field `near_clip_plane` - `near` for descriptiveness, `clip` for brevity (and the type then `ClipPlane`)

(?)